pkg/volume/emptydir/empty_dir_quota.go

```diff
@@ -69,10 +69,16 @@ func (edq *emptyDirQuotaMounter) SetUp(fsGroup *int64) error {
 
 func (edq *emptyDirQuotaMounter) SetUpAt(dir string, fsGroup *int64) error {
 	err := edq.wrapped.SetUpAt(dir, fsGroup)
-	if err == nil {
-		err = edq.quotaApplicator.Apply(dir, edq.medium, edq.pod, fsGroup, edq.quota)
+	if err != nil {
+		return err
 	}
-	return err
+
+	err = edq.quotaApplicator.Apply(dir, edq.medium, edq.pod, fsGroup, edq.quota)
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
Should we just return "err" here to be a little more concise and drop the "if" above? 

Could you link to the style consistency you refer to here for my own knowledge?